cloudformation-states.yaml

```diff
@@ -0,0 +1,110 @@
+                    "QueryParameters": {
+                      "fieldSelector": [
+                        "status.successful=1"
+                      ]
+                    }
+                  },
+                  "Next": "LaunchKubernetesJobIfNotExists"
+                },
+                "LaunchKubernetesJobIfNotExists": {
+                  "Type": "Task",
+                  "Comment": "Attempts to create a Kubernetes Job for the pipeline server. Will swallow a 409 (already exists) error.",
+                  "Resource": "arn:aws:states:::eks:call",
+                  "Parameters": {
+                    "ClusterName": "biomage-${Environment}",
+                    "CertificateAuthority.$": "$.certificateAuthority",
+                    "Endpoint": "${Endpoint}",
+                    "Method": "POST",
+                    "Path": "/apis/batch/v1/namespaces/default/jobs",
+                    "RequestBody": {
+                      "apiVersion": "batch/v1",
+                      "kind": "Job",
+                      "metadata": {
+                        "name": "remoter-server-${ExperimentId}"
+                      },
+                      "spec": {
+                        "template": {
+                          "metadata": {
+                            "name": "remoter-server-${ExperimentId}"
+                          },
+                          "spec": {
+                            "containers": [
+                              {
+                                "name": "remoter-server",
+                                "image": "242905224710.dkr.ecr.eu-west-1.amazonaws.com/pipeline:refs-heads-master-236bab58afda4a763bc6a72514247e228fae5c75-remoter-server"
+                              }
+                            ],
+                            "restartPolicy": "Never"
+                          }
+                        }
+                      }
+                    }
+                  },
+                  "Retry": [
+                    {
+                      "ErrorEquals": [ "EKS.409" ],
+                      "IntervalSeconds": 1,
+                      "BackoffRate": 2.0,
+                      "MaxAttempts": 2
+                    }
+                  ],
+                  "Catch": [
+                    {
+                      "ErrorEquals": [ "EKS.409" ],
+                      "ResultPath": "$.error-info",
+                      "Next": "Wait"
+                    }
+                  ],
+                  "Next": "Wait"
+                },
+                "Wait": {
+                  "Type": "Wait",
+                  "Seconds": 5,
+                  "End": true
+                }
+              }
+            }
+          - Endpoint:
+              Fn::ImportValue:
+                !Sub "eksctl-biomage-${Environment}-cluster::Endpoint"
+              
+      RoleArn:
+        Fn::ImportValue:
+          !Sub "biomage-state-machine-role-${Environment}::ARN"
```

Comment:
This is an ARN role that is created in `iac`. This will probably need to be changed for the production version as well, but this way it's a standalone CF file that can work.